hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientMembershipListener.java

```diff
@@ -162,9 +162,7 @@ private void memberRemoved(Member member) {
         logger.info(membersString());
         final Connection connection = connectionManager.getConnection(member.getAddress());
         if (connection != null) {
-            connectionManager.destroyConnection(connection,
-                    null,
-                    newTargetDisconnectedExceptionCausedByMemberLeftEvent(member, connection.toString()));
+            connection.close(null, newTargetDisconnectedExceptionCausedByMemberLeftEvent(member, connection.toString()));
```

Comment:
There is not a change. The close of the connection is always done on the connection; the connection then will do a callback to the connection manager to remove whatever needs to be removed. Otherwise you have 2 api's to close a connection; this has been simplified to a single API.